Fab/Fab.jsx

```diff
@@ -11,13 +12,23 @@ class Fab extends MaterialComponent {
     super();
     this.componentName = "fab";
     this._mdcProps = ["mini", "exited"];
+    this.themeProps = ["primary", "secondary"];
   }
   componentDidMount() {
     super.attachRipple();
   }
   materialDom(props) {
+    let className = "";
```

Comment:
`= props.class || props.className`

Reply:
also to my horror we missed it in button as well, can you add it there as well?

Reply:
It should actually be `= props.class || props.className || ""` because it would otherwise be undefined if `class` or `className` are not defined

Reply:
And then I would need to move `{...props}` before `class={className}`, correct?

Reply:
We should actually omit `class` and `className` from {...props} but the order doesn't really matter. See https://github.com/prateekbh/preact-material-components/blob/master/Select/Select.jsx#L112 for how to omit the properties (please note that I clone the props as to be certain that I don't modify the actual props).